src/test-routes/security.ts

```diff
@@ -0,0 +1,19 @@
+import { app, json } from "../api";
+
+app.category("vanilla", () => {
+  app.head("/securitykey", "SecurityKey", (req) => {
+    req.expect.containsHeader("security-key", "123456789");
+
+    return {
+      status: 200,
+    };
+  });
+});
+
+app.category("vanilla", () => {
```

Comment:
I think it would be better to include the AAD test under the azure category

Reply:
For `vanilla`, I think it is also possible for AAD(like "webpubsub")?